CONTRIBUTING.md

```diff
@@ -167,50 +204,104 @@ make license-check
 
 Note: you need to define the `GITHUB_TOKEN` environment variable to a valid GitHub personal access token, or you will hit rate limiting from the GitHub API which `licensei` uses to discover the licenses of dependencies.
 
-## Acceptance Tests: Testing interactions with external services
+---
 
-OpenTofu's unit test suite is self-contained, using mocks and local files to help ensure that it can run offline and is unlikely to be broken by changes to outside systems.
+### Backporting
 
-However, several OpenTofu components interact with external services.
+All changes to OpenTofu should, by default, go into the `main` branch. When a fix is important enough, it will be backported to a version branch and release when the next minor release rolls around.
 
-There are some optional tests in the OpenTofu CLI codebase that *do* interact with external services, which we collectively refer to as "acceptance tests". You can enable these by setting the environment variable `TF_ACC=1` when running the tests. We recommend focusing only on the specific package you are working on when enabling acceptance tests, both because it can help the test run to complete faster and because you are less likely to encounter failures due to drift in systems unrelated to your current goal:
+Start the backporting process by making sure both the `main` and the target version branch are up-to-date in your working copy. Look up the commit ID of the commit and then switch to your target version branch. Now create a new branch from that version branch:
 
+```sh
+git checkout -b backports/ISSUE_NUMBER
 ```
-TF_ACC=1 go test ./internal/initwd
+
+Now you can cherry-pick the commit in question to your backport branch:
+
+```sh
+git cherry-pick -s COMMIT_ID_HERE
 ```
 
-Because the acceptance tests depend on services outside of the OpenTofu codebase, and because the acceptance tests are usually used only when making changes to the systems they cover, it is common and expected that drift in those external systems will cause test failures. Because of this, prior to working on a system covered by acceptance tests it's important to run the existing tests for that system in an *unchanged* work tree first and respond to any test failures that preexist, to avoid misinterpreting such failures as bugs in your new changes.
+Finally, create an additional commit to edit the changelog and send in your PR. 
 
-### Integration Tests: Testing interactions with external backends
+---
 
-OpenTofu supports various [backends](https://opentofu.org/docs/language/settings/backends/configuration). We run integration test against them to ensure no side effects when using OpenTofu.
+## FAQ
```

Comment:
I think this FAQ definitely needs a separate document of its own. 
How about moving this part to a separate document and linking it here? 

I think when the Contribution document gets bigger and longer, people might refrain reading it.

Reply:
Oh, also, TOC please for FAQ's, or cascading questions to prevent doom scrolling through FAQ document 😅

Reply:
I'm a bit split on ToCs, GitHub already has them auto-generated for markdown files, but their usability is... less than ideal.